docs/source/ui_components.rst

```diff
@@ -0,0 +1,32 @@
+UI Components
+=============
+When opening bug reports or sending emails to the Jupyter mailing list, it is
+useful to know the names of different UI component so that other developers
+and users have an easier time helping you diagnose your problems. This section
+of the documentation is designed to familiarize you with the different names
+of UI elements and the different states that UI elements can be in.
```

Comment:
Let's tighten this up.

Perhaps: This section will familiarize you the names of UI elements and the different Notebook modes (i.e. editing, command mode).